src/github/deployment.ts

```diff
@@ -30,11 +30,13 @@ export const deploymentWebhookHandler = async (context: WebhookContext, jiraClie
 		webhookPayload: context.payload,
 		webhookReceived: Date.now(),
 		webhookId: context.id,
+		entityAction: context.action,
```

Comment:
You may not need this line. Coz we can juse use the deployment.state within the jiraPayload